src/applications/static-pages/facilities/FacilityDetailWidget.jsx

```diff
@@ -21,29 +22,52 @@ export class FacilityDetailWidget extends React.Component {
     // Sort and compile facility hours into a list
     const hours = facilityDetail.attributes.hours;
     const builtHours = buildHours(hours, true);
+    const clinicalHours = builtHours.map((day, index) => {
+      const splitDay = day.split(': ');
+      const abbrvDay = splitDay[0];
+      const times = splitDay[1];
+
+      let el = '';
+      if (index === 0 || index === 5) {
+        el += `<ul class="vads-u-flex--1 va-c-facility-hours-list vads-u-margin-top--0 ${
+          index === 0
+            ? 'vads-u-margin-bottom--1 small-screen:vads-u-margin-bottom--0 vads-u-margin-right--3'
+            : 'vads-u-margin-bottom--0'
+        }">`;
+      }
+
+      el += `<li key=${index}>
+                <b class="abbrv-day">${abbrvDay}:</b> ${times}
+              </li>`;
+
+      if (index === 4 || index === 6) {
+        el += '</ul>';
+      }
+
+      return el;
+    });
 
     return (
       <div key={facilityDetail.id} className="vads-c-facility-detail">
         <section className="vads-facility-detail">
-          <FacilityAddress facility={facilityDetail} />
+          <h3 className="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">
+            Address
+          </h3>
+          <div className="vads-u-margin-bottom--3">
+            <FacilityAddress facility={facilityDetail} />
+          </div>
+          <h3 className="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">
+            Phone numbers
+          </h3>
           <FacilityPhone facility={facilityDetail} />
           <div className="vads-u-margin-bottom--0">
             <div className="clinicalhours">
               <h3 className="vads-u-margin-top--2p5 vads-u-margin-bottom--1">
-                Clinical Hours
+                Clinical hours
               </h3>
-              <ul className="va-c-facility-hours-list vads-u-margin-top--0">
-                {builtHours.map((day, index) => {
-                  const splitDay = day.split(': ');
-                  const abbrvDay = splitDay[0];
-                  const times = splitDay[1];
-                  return (
-                    <li key={index}>
-                      <b className="abbrv-day">{abbrvDay}:</b> {times}
-                    </li>
-                  );
-                })}
-              </ul>
```

Comment:
What had to change here? I think the old way is cleaner and more idiomatic for React.

Reply:
they wanted monday tuesday wednesday thursday and friday in one column and sun and sat in another column. the only way to do this was to manually put in the `<ul>` tags in the right places. before, we had css splitting the two columns, but it didn't do it correctly - it had friday in the saturday and sunday column because it tries to split the items evenly - 4 items in one column and 3 items in the other. 

if you know how to do this without manually putting in the `<ul>` tags, please let me know. i would love to learn